packages/@okta/vuepress-site/docs/concepts/scim/index.md

```diff
@@ -563,44 +563,6 @@ Whether Partner-Built EA or Okta-Verified, when issues arise related to the SCIM
 
```

Comment:
Not part of your changes, I know, but in Line 125 there's a broken link to devforum.

Reply:
Also found it at line 261. Fixed both.